my/src/views/Status.vue

```diff
@@ -1,294 +1,63 @@
-              <v-chip
-                style="border:none;float:left;overflow:wrap;"
-                outline
-                small
-                color="#555"
-              >
-                <v-icon left>check</v-icon>
-                <strong>Your response has been submitted.</strong>
-              </v-chip>
-            </div>
-            <!-- <div class="rspvButtons">
-                            <v-btn class="button1" @click="submitRSVP">Submit</v-btn>
-            </div>-->
-          </div>
-        </div>
-        <div class="wrap-status100" v-if="step <= 3">
-          <div>
-            <h1
-              v-show="!genderCompleted"
-              slot="activator"
-              style="color: #F14D4C"
-            >
-              Please go to
-              <a href="/apply" class="currentStatus">your application</a>
-              and complete an additional field.
-            </h1>
-            <h1 v-show="step === 0 && genderCompleted">
-              You haven't started yet! Go
-              <a href="/apply" class="currentStatus">here</a>
-              to begin.
-            </h1>
-            <h1 v-show="step > 0 && genderCompleted">{{ currentHeader }}</h1>
-            <br />
-            <v-stepper alt-labels class="transp">
-              <v-stepper-header>
-                <v-stepper-step step="1" :complete="step > 0">
-                  {{ baseStep }}
-                </v-stepper-step>
-                <v-divider></v-divider>
-                <v-stepper-step step="2" :complete="step > 0">
-                  Submitted
-                </v-stepper-step>
-                <v-divider></v-divider>
-                <v-stepper-step step="3" :complete="step > 0">
-                  Processing
-                </v-stepper-step>
-                <v-divider></v-divider>
-                <v-stepper-step step="4" :complete="step > 0">
-                  Decision
-                </v-stepper-step>
-              </v-stepper-header>
-            </v-stepper>
+    <Navbar2 />
+      <div class="wrap">
+        <!--Column#1-->
+        <div class="col col6">
+          <div class="box box9">
+            <p class="big">Welcome.</p>
+            <p class="small">Be a part of the hackathon for change.
+              We are looking forward to being able to meet with you in person.
+              Good luck with your application!</p>
           </div>
-        </div>
-        <br />
-        <br />
-        <br />
-        <br />
-        <div>
-          <v-layout row wrap>
-            <v-flex d-flex md3 :key="media.icon" v-for="media in social">
-              <div>
-                <v-btn class="mx-3" dark icon :href="media.link">
-                  <v-icon size="24px">{{ media.icon }}</v-icon>
-                </v-btn>
+          <div class="box box5 status">
+            <div class="currentStatus">
+              <h2>My Application Status</h2>
+              <v-icon> {{ emoticon }}</v-icon>
```

Comment:
Not sure but don't think this is defined anywhere

Reply:
might be getting passed as a prop?